api/certs.yml

```diff
@@ -58,6 +57,24 @@ paths:
             Failed to revoke corresponding certificate.
         '500':
           $ref: "#/components/responses/ServiceError"
+  /serials/{thingID}:
+    get:
+      summary: Retrieves certificates's serial IDs
```

Comment:
It's probably a good idea to use a neutral form here: `Retrieve certificates serial IDs`. The same goes for all the other `summary` sections.

Reply:
We are using this form in all other api docs. Should we change all?

Reply:
Probably yes, but let's do it in a separate PR.